pre_commit_hooks/go/goimports.sh

```diff
@@ -2,4 +2,6 @@
 
 set -e
 
-golangci-lint run "$@"
+for file in "$@"; do
+    goimports -l -w "$(dirname "$file")"
+done
```

Comment:
actually saw this... but thought this might just replace it for compatibility ;)

Reply:
Nope lol! At some point we will replace goimports with golangci-lint but I need to find time for writing a proper configuration before.